### PR TITLE
add missing fan info to dumps

### PR DESCRIPTION
**Pull request by ChaosInfinited**
[Issue](https://github.com/Umio-Yasuno/amdgpu_top/issues/83)

Added max fan speed to json dump and current fan speed to normal dump.
I haven't been coding in Rust, but I hope this was simple enough for me not to butcher it :)

### CODE REVIEW

File: docs/dump.json [hunk 1]
```diff
@@ -68,6 +68,10 @@
         "unit": "RPM",
         "value": 0
       },
+      "Fan Max": {
+        "unit": "RPM",
+        "value": 6000
```
Comment (ChaosInfinited) [inline docs/dump.json:73]: Saw you're defaulting it to 6k in here:
https://github.com/Umio-Yasuno/amdgpu_top/blob/ee770d7a6982647b1a9702cd2f30c384d3ce0eee/crates/amdgpu_top_gui/src/app.rs#L961

Comment (Umio-Yasuno) [reply docs/dump.json:73]: Do not update `docs/*.json` in the PR as it will be regenerated later in my environment.

Comment (ChaosInfinited) [reply docs/dump.json]: I see, sorry, reverted it.

**Comment by Umio-Yasuno** [comment]
Merge this PR. Thanks!

**Pull request merged**